Rakefile

```diff
@@ -2,82 +2,90 @@ require 'rspec/core/rake_task'
-  generate += " #{target}"
-  generate += " > acceptance/hosts.cfg"
-  sh generate
-  sh "cat acceptance/hosts.cfg"
-end
+  task :unit => [:check_unit]
 
-default_rake_ver = '11.0'
-
-rototiller_task :acceptance => [:generate_host_config] do |t|
-  t.add_env({:name => 'LAYOUT',   :default => 'centos7-64',
-             :message => 'The argument to pass to beaker-hostgenerator',
-             :set_env => true})
-  t.add_env({:name => 'RAKE_VER', :default => default_rake_ver,
-             :message => 'The rake version to use when running unit and acceptance tests',
-             :set_env => true})
-
-  t.add_flag do |flag|
-    flag.name = '--hosts'
-    flag.default = 'acceptance/hosts.cfg'
-    flag.message = 'The configuration file that Beaker will use'
-    flag.override_env = 'BEAKER_HOSTS'
-  end
-  t.add_flag do |flag|
-    flag.name = '--preserve-hosts'
-    flag.default = 'onfail'
-    flag.message = 'The beaker setting to preserve a provisioned host'
-    flag.override_env = 'BEAKER_PRESERVE_HOSTS'
-  end
-  t.add_flag do |flag|
-    flag.name = '--keyfile'
-    flag.default ="#{ENV['HOME']}/.ssh/id_rsa-acceptance"
-    flag.message = 'The SSH key used to access a SUT'
-    flag.override_env = 'BEAKER_KEYFILE'
-  end
-  t.add_flag do |flag|
-    flag.name = '--load-path'
-    flag.default = 'acceptance/lib'
-    flag.message = 'The load path Beaker will use'
-    flag.override_env = "BEAKER_LOAD-PATH"
-  end
-  t.add_flag do |flag|
-    flag.name = '--pre-suite'
-    flag.default = 'acceptance/pre-suite'
-    flag.message = 'THe path to a directory containing pre-suites'
-    flag.override_env = "BEAKER_PRE-SUITE"
+  default_rake_ver = '11.0'
+
+  desc "" # empty description so it doesn't show up in rake -T
+  rototiller_task :check_unit do |t|
+    t.add_env({:name => 'SPEC_PATTERN', :default => 'spec/', :message => 'The pattern RSpec will use to find tests'})
+    t.add_env({:name => 'RAKE_VER',     :default => default_rake_ver,  :message => 'The rake version to use when running unit tests'})
   end
-  t.add_flag do |flag|
-    flag.name = '--tests'
-    flag.default = 'acceptance/tests'
-    flag.message = 'The path to the tests you want beaker to run'
-    flag.override_env = 'BEAKER_TESTS'
+
+  desc ""
+  #FIXME: this is probably a build task, given that it has an output file
+  rototiller_task :generate_host_config do |t|
+    t.add_command do |bhg_command|
+      bhg_command.name = "beaker-hostgenerator"
+      bhg_command.add_argument do |argument|
+        # ugh, this is terrible, but it's the only way in current rototiller
+        argument.name = ENV['BEAKER_ABS'] ? ' --hypervisor=abs' : ''
+        argument.message = 'Use hypbervisor abs option for bhg'
+      end
+      bhg_command.add_argument do |arg|
+        arg.name = 'centos7-64'
+        arg.add_env({:name => 'LAYOUT', :message => 'The beaker-hostgenerator pattern'})
+        arg.add_env({:name => 'TEST_TARGET', :message => 'The beaker-hostgenerator pattern'})
```

Comment:
What is the difference between LAYOUT and TEST_TARGET? Should they have unique descriptions?